plan.md

```
@@ -0,0 +1,423 @@
+- `~/code/projects/github-json-bourne` — the Action itself (new repo, already created)
+- `~/code/projects/gwilym.ca` — the consumer (update workflow + enhance components)
+
+---
+
+## Phase 1: Bootstrap the Action repo
+
+**Goal:** Scaffold `github-json-bourne` as a proper Node.js GitHub Action.
+
+**Files to create in `~/code/projects/github-json-bourne`:**
+
+### 1.1 `action.yml` — Action metadata
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)